content/post/community-india-is-back.md

```diff
@@ -1,10 +1,50 @@
+description = "AWS Community Days are community-organized cloud education events, featuring technical discussions and demos led by expert AWS users and industry leaders from around the world."
 feature_image = ""
-tags = []
+tags = ["AWS"]
 title = "Welcome to AWS Community Day India, Virtual Edition"
 
 +++
-Welcome back! We're glad to have Community Day going again.
\ No newline at end of file
+We are glad to announce **AWS Community Day 2020, India Virtual Edition** from **16th - 17th October 2020** being organized by various AWS User Groups in India.
+
+![](/images/ug-logo-for-profile-pic-dark.png)
+
+**AWS User Group India** is united by community-led learning and diversified by the wide variety of tools and technologies closely knit with Amazon Web Services, cloud computing technology, AWS design implementation and servicing, high scalability / performance computing, production use cases of AWS, block chain, Artificial Intelligence and much more.
+
+Community-based learning is the best way to enhance the concepts being taught by connecting with enthusiasts in person, thereby sharing first-hand experiences and familiar, accessible examples.
+
+All skill levels are welcome to help accomplish our motto of learning, sharing and networking. Let's join hands to empower this tech community to make it bigger and better!
+
+## **WHAT TO EXPECT FROM AWS COMMUNITY DAY?**
+
+* Renew, revive & revitalize to exemplify your excitement about the work you do.
+* Learn industry trends and intricacies about innovative implementations from AWS experts.
+* Harness the power of networking with AWS users, DevOps, developers & solutions architects.
+* Network with fellow AWS veterans & techies during breaks.
+* Novice to pioneer on relentless pursuit to learn can rejoice the AWS offerings & success stories under one roof.
+* Take helpful and valuable information back to business.
+* Gain visibility to get more publicity for your esteemed organisation.
+* Sporting chance for active participants to win AWS goodies.
+
+Please go through the [Code of Conduct](https://communityday.awsug.in/code-of-conduct.html) of the event.
+
+## **WORKSHOPS**
+
+AWS Community Day 2020 comes with 4 hands-on workshops listed as below:
+
+* **Containers:** The workshop will be lead by Sandip Das ( AWS Container Hero) and Dipali Kulshrestha (AWS Community Leader) on 17th October at 2:00 pm IST. The workshop will be of 4 hours.
+* **Machine Learning:** The workshop will be lead by Jayesh Ahire (AWS ML Hero) and Sukanya Mandal (AWS reInvent 2019 Diversity Scholarship Winner) on 17th October at 2:00 pm IST. The workshop will be of 4 hours.
+* **Serverless:** The workshop will be lead by Bhuvaneswari Subramani and Dhaval Nagar on 17th October at 2:00 pm IST. The workshop will be of 4 hours.
+* **Alexa:** The workshop will be lead by Karthik Ragubathy(Solutions Architect) with the Alexa team on 17th October at 2:00 pm IST. The workshop will be of 4 hours
+
+  ## **KEY-NOTE SPEAKER**
```

Comment:
should be `Keynote`

Reply:
Done. Thank you for correcting @SathyaBhat.